backend/app/services/first_run_doctor.py

```
@@ -500,27 +512,163 @@ def _source_governance_unavailable_checks(
     ]
 
 
-def _check_backend_expectation(backend_base_url: str) -> FirstRunDoctorCheck:
-    normalized = backend_base_url.rstrip("/")
+def _normalize_base_url(base_url: str) -> str:
+    return base_url.rstrip("/")
+
+
+def _http_get(url: str) -> HttpProbeResponse:
+    request = Request(url, headers={"Accept": "application/json, text/html"})
```

Ruff (0.15.11)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)